conformance/lang/expressions/shift-expression/left_shift.balt

```diff
@@ -0,0 +1,3816 @@
+Labels: DecimalFloatingPointNumber, DecimalTypeSuffix, FloatingPointTypeSuffix, int, shift-expr, type-cast-expr
+
+function init() {
+    int a = <int> 6.0 << <int> 7d;
+    io:println(a); // @output 768
+}
+
+Test-Case: output
+Description: Test '<<' expression with unary expressions as operands.
+Labels: DecimalNumber, int, shift-expr, unary-complement, unary-minus, unary-plus
+
+function init() {
+    int a = +4 << -3;
+    io:println(a); // @output -9223372036854775808
+
+    int b = +4 << +3;
+    io:println(b); // @output 32
+
+    int c = -(-2 << 5);
+    io:println(c); // @output 64
+
+    int d = ~(2 << 5);
+    io:println(d); // @output -65
+
+    int e = ~2 << 5;
+    io:println(e); // @output -96
+}
+
+Test-Case: output
+Description: Test '<<' expression with multiplicative expressions operands.
+Labels: DecimalNumber, int, multiplicative-expr, shift-expr
+
+function init() {
+    int a = 2 * 2 << 404 % 10 / 4;
+    io:println(a); // @output 8
+}
+
+Test-Case: output
+Description: Test '<<' expression additive expressions as operands.
+Labels: additive-expr, DecimalNumber, int, shift-expr
+
+function init() {
+    int a = (1 + 2) << (3 + 4);
+    io:println(a); // @output 384
+}
+
+Test-Case: output
+Description: Test '<<' expression with shift expressions as operands.
+Labels: DecimalNumber, int, shift-expr
+
+function init() {
+    int a = (1 << 2) << (100 >> 4);
+    io:println(a); // @output 256
+}
+
+Test-Case: output
+Description: Test '<<' expression with binary bitwise expressions as operands.
+Labels: binary-bitwise-expr, DecimalNumber, int, shift-expr
+
+function init() {
+    int a = (2 & 6) << (2 ^ 6);
+    io:println(a); // @output 32
+}
+
+Test-Case: output
+Description: Test '<<' expression with conditional expressions as operands.
+Labels: conditional-expr, DecimalNumber, error, int, nil-literal, optional-type, shift-expr, union-type
+
+function init() {
+    int|error a = 12;
+    int? b = ();
+    int c = (a is error ? 2 : a) << (b ?: 13);
+    io:println(c); // @output 98304
+    int d = (b ?: 13) << (a is error ? 2 : a);
+    io:println(d); // @output 53248
+}
+
+Test-Case: output
+Description: Test '<<' expression with checking expressions as operands.
+Labels: checking-expr, DecimalNumber, error-type, int, shift-expr, union-type
+
+function init() returns error? {
+    int|error a = 12;
+    int b = check a << checkpanic a;
+    io:println(b); // @output 49152
+
+    int c = checkpanic a << check a;
+    io:println(c); // @output 49152
+}
+
+Test-Case: output
+Description: Test valid '<<' expression.
+Labels: DecimalNumber, int, shift-expr, unary-minus
+
+function init() {
+    int a = 12<<2;
+    io:println(a); // @output 48
+    int b = -12<<-2;
+    io:println(b); // @output 0
+    int c = 12  <<  2;
+    io:println(c); // @output 48
+    int d = 32 <<
+    1;
+    io:println(d); // @output 64
+    int e = 32
+    << 2;
+    io:println(e); // @output 128
+}
+
+Test-Case: parser-error
```

Comment:
Lest move this test to the top

Reply:
Addressed in https://github.com/ballerina-platform/ballerina-spec/pull/953/commits/35b7604ebe9005245b1a957701561a60f12f3a1a.